### PR TITLE
Resolved #27: Added PushParser helper class

### LINKED ISSUE #27: Add method for push message parsing
We should provide a helper method for verifying that a push message is a "mtoken" one.

### PR DESCRIPTION
**Pull request by kober32**
Resolved issue #27.

### CODE REVIEW

File: docs/Using-Push-Service.md [hunk 1]
```diff
@@ -66,11 +76,32 @@ FirebaseInstanceId.getInstance().instanceId.addOnCompleteListener { task ->
...
+_Example push notification processing:_
+
+```kotlin
+// Overriden method of FirebaseMessagingService
+override fun onMessageReceived(remoteMessage: RemoteMessage) {
+    super.onMessageReceived(remoteMessage)
+    val push = PushParser.parseNotification(remoteMessage.data)
+    if (push != null) {
+        // process the notification and react to it in the UI
+    }
```
Comment (petrdvorak) [inline docs/Using-Push-Service.md:105]: Add info about the fact that this is the place where the rest of the processing should happen, i.e.:

```kotlin
    // process the mtoken notification and react to it in the UI
} else {
    // process all the other notification types using your own logic
}

File: library/src/main/java/com/wultra/android/mtokensdk/push/PushParser.kt [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+         * @return parsed known [PushMessage] or null
+         */
+        @JvmStatic
+        @JvmName("parseNotification")
+        fun parseNotification(notificationData: Map<String, String>): PushMessage? {
+
+            val id = notificationData["operationId"] ?: return null
+            val name = notificationData["operationName"] ?: return null
+
+            return when (notificationData["messageType"]) {
```
Comment (petrdvorak) [inline library/src/main/java/com/wultra/android/mtokensdk/push/PushParser.kt:36]: The method should also return any other data that push message sender decides to include in the payload, for example as an `extras` attributes.

Comment (kober32) [reply library/src/main/java/com/wultra/android/mtokensdk/push/PushParser.kt:36]: Yea, I added an `originalData` property to it.

File: docs/Using-Push-Service.md (hunk 1 above)
Comment (kober32) [reply docs/Using-Push-Service.md:109]: 👍

**Pull request merged**